install/kubernetes/cilium/values.yaml

```diff
@@ -792,8 +792,10 @@ localRedirectPolicy: false
 # labels: ""
 
 # logOptions allows you to define logging options. eg:
-# logOptions:
-#   format: json
+# logOptions: |
```

Comment:
agreed this one is clearer. Do we need to update the internal convertion in config map ?

https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/templates/cilium-configmap.yaml#L516-L518

Reply:
oh right that would be better!

Reply:
actually as it work like this I believe we can leave it as is 🤔

Reply:
While changing the format is more user friendly, it might cause a little hiccup for 1.9 users (e.g. helm upgrade with --reuse-values flag).

I am not sure how many users are using this option in 1.9 though :thinking:

Reply:
Isn't it more difficult to override/set `logOptions` through the helm cli after this patch?

For example, before this patch I can write:
```
helm template ... \
  --set logOptions.format=json \
  --set logOptions.level=debug
```
But after this patch I have to write JSON "by hand":
```
helm template ... \
  --set logOptions='{"format":"json","level":"debug"}'
```

Reply:
Oh right, I lean toward to declarative configuration (e.g. passing values.yaml), but the above is another convincing reason to keep it as is. 

What do you think @mvisonneau ? :thinking:

Reply:
absolutely, my bad guys, I gave it another try and indeed, it already works as expected! I was under the impression that @michi-covalent's comment was still valid: https://github.com/cilium/cilium/pull/11861#issuecomment-702986240

Let's close this PR!